apps/codeforafrica/src/components/OpportunityCardList/OpportunityCardList.js

```diff
@@ -5,15 +5,19 @@ import OpportunityCard from "@/codeforafrica/components/OpportunityCard";
 
 const OpportunityCardList = React.forwardRef(
   function OpportunityCardList(props, ref) {
-    const { opportunities, ...other } = props;
+    const { opportunities, readMore, ...other } = props;
 
     if (!opportunities?.length) {
       return null;
     }
     return (
       <Stack spacing={{ xs: 5, md: 10 }} {...other} ref={ref}>
         {opportunities.map((opportunity) => (
-          <OpportunityCard {...opportunity} key={opportunity.id} />
+          <OpportunityCard
+            {...opportunity}
+            key={opportunity.id}
```

Comment:
```suggestion
            key={opportunity.slug}
```
slug, href should be unique, no?